Error: Cannot find module './dist/lib/generator'

Version: `v1.4.0`

Ok i'm sorry, here is :+1: 
module.js:549
    throw err;
    ^

Error: Cannot find module './dist/lib/generator'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/home/boisseleau/.nvm/versions/node/v8.11.2/lib/node_modules/generate-swap-project/generator.js:1:80)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)

--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "generate-swap-project",
   "description": "Generate An ES.Next/StandardJS/UnitTest Ready, Github or Gitlab Project in Seconds! − A generator for SWAP projects −",
-  "version": "1.4.0",
+  "version": "1.4.1",
   "homepage": "https://github.com/sirap-group/generate-swap-project",
   "author": "Rémi Becheras (https://github.com/rbecheras)",
   "repository": "sirap-group/generate-swap-project",

